public/js/chat.js

```diff
@@ -38,6 +41,27 @@ var Chat = function(id, player) {
 		});
 	});
 
+	//pulling the emote json file from the dgg cdn, not sure how to do this without a cors proxy :(
```

Comment:
You could use the server-side to grab the emotes.json and then serve it to the client, rather than doing it this way